src/lib.zig

```diff
@@ -5,6 +5,10 @@ pub const c = @cImport({
     @cInclude("TracyC.h");
 });
 
+pub fn frame() callconv(.Inline) void {
+    c.___tracy_emit_frame_mark("Frame");
```

Comment:
can you expose this as a parameter?

Reply:
Will do when I understand if it should be an optional type or not like with `trace`: https://github.com/nektro/zig-tracy/pull/2#discussion_r748720849

Reply:
Done!